corehq/apps/geospatial/utils.py

```diff
@@ -7,3 +14,46 @@ def get_geo_case_property(domain):
     except GeoConfig.DoesNotExist:
         config = GeoConfig()
     return config.case_location_property_name
+
+
+def get_geo_user_property(domain):
+    try:
+        config = GeoConfig.objects.get(domain=domain)
+    except GeoConfig.DoesNotExist:
+        config = GeoConfig()
+    return config.user_location_property_name
+
+
+def _format_coordinates(lat, lon):
+    return f"{lat} {lon} 0.0 0.0"
+
+
+def process_gps_values_for_case(domain, case_data):
+    location_prop_name = get_geo_case_property(domain)
+    helper = CaseHelper(domain=domain, case_id=case_data['id'])
+    case_data = {
+        'properties': {
+            location_prop_name: _format_coordinates(case_data['lat'], case_data['lon'])
+        }
+    }
+
+    helper.update(case_data)
+
+
+def process_gps_values_for_user(domain, user_data):
```

Comment:
Point 1 of [this comment](https://github.com/dimagi/commcare-hq/pull/33420/files#r1321426882) is also applicable here